task1/src/tasks/task1.ts

```diff
@@ -0,0 +1,29 @@
+const readStdinStream = process.stdin;
+const writeStdoutStream = process.stdout;
+
+export function reverseStringFromStdin(str: string): string {
+  const stringWithoutCRLF = str.slice(0, str.length - 2);
+  const CRLF = str.slice(-2);
+  const reversedString = stringWithoutCRLF.split('').reverse().join('');
+
+  return `${reversedString}${CRLF}`;
+}
+
+export function task1() {
```

Comment:
1. Always use arrow functions: `const task1 = () => {...}`
2. _task1_ is not proper naming. Please follow clean code and give consistent names